app/src/main/java/com/dodo/flashcards/domain/models/flashcard/Flashcard.kt

```diff
@@ -0,0 +1,9 @@
+package com.dodo.flashcards.domain.models.flashcard
+
+class Flashcard : IFlashcard{
+	override val front: String
+		get() = TODO("Not yet implemented")
+
+	override val back: String
+		get() = TODO("Not yet implemented")
+}
```

Comment:
We don't need this class for the future since our data layer will be returning an anonymous object `object : Flashcard { }`

for mocking, this will be more beneficial and cleaner if you make this a data class 
ie `data class FlashcardImpl(override val front: String, override val back: String) : Flashcard`
https://kotlinlang.org/docs/data-classes.html